cli/src/commands/auth.rs

```diff
@@ -27,12 +27,34 @@ async fn handle_auth_login(mut config: Config, config_path: &Path) -> CommandRes
 }
 
 /// Display the current authentication status to the user.
-pub fn handle_auth_status(config: &Config) {
-    if config.auth_info.offline_access.is_some() {
-        print_user_success!("Currently authenticated with long lived refresh token");
-    } else {
-        print_user_warning!("User is not currently authenticated");
+pub async fn handle_auth_status(
+    mut config: Config,
+    timeout: Option<u64>,
+    ignore_certs: bool,
+) -> CommandResult {
+    // Create a client with our auth token attached.
+    let api = PhylumApi::new(
```

Comment:
Hm that's a good point. I was wondering if I should make use of API at all or side-step it and instead try to manually send a request. But I didn't want to duplicate any code.

Would you suggest sending the Request without the help of `PhylumApi` or instead maybe just check first if there is a token and early-return in case one is not present yet.

Alternatively a parameter could be added to `PhylumApi` to perform auth, but I don't really like that myself.